pages/app/Explore.js

```diff
@@ -516,14 +551,10 @@ class Explore extends Page {
     const { geographiesTree, topicsTree, dataTypeTree, zoom, latLng } = explore;
     const { showFilters } = this.state;
     const { topics, geographies, dataType } = this.filters;
-    const topicsLabels = topics.map(topic => findTagInSelectorTree(topicsTree, topic).label);
-    const geographiesLabels = geographies.map(geography =>
-      findTagInSelectorTree(geographiesTree, geography).label);
-    const dataTypeLabels = dataType.map(dType => findTagInSelectorTree(dataTypeTree, dType).label);
 
-    const allTagsSt = [].concat(topicsLabels).concat(geographiesLabels)
-      .concat(dataTypeLabels).join(', ');
-    const filtersSumUp = !showFilters && allTagsSt.length > 0 ? `Filtering by ${allTagsSt}` : '';
+    const selectedTags = [...topics.map(t => ({ ...findTagInSelectorTree(topicsTree, t), type: 'topics' })),
```

Comment:
I think this is the perfect example to create a component. Don't you feel overwhelmed when you enter Explore page and see 878 lines of code? We should avoid loading everything on the page. That's why we need to create components.

Here you have an example => https://github.com/resource-watch/resource-watch/tree/develop/components/dashboards/wysiwyg/widget-block-edition